### PR TITLE
Update pre-requisite for executing on Windows 10 and Windows 11

### PR DESCRIPTION
**Pull request by vrukesh**
Update pre-requisite for executing on Windows 10 and Windows 11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ Download a release of the toolchain for your platform from [Github
 releases](https://github.com/ARM-software/LLVM-embedded-toolchain-for-Arm/releases)
 and extract the archive into an arbitrary directory.
 
+### Pre-requisite to use toolchain in Windows 10 and Windows 11
```
Comment (voltur01) [inline README.md:76]: "to using the toolchain" to be consistent with the next title?

Comment (voltur01) [inline README.md:76]: It looks like "on Windows" is more commonly used?

Comment (voltur01) [inline README.md:76]: Replace with just Windows (without 10 and 11 - list of supported Windows versions should be elsewhere, not need to copy it here).

File: README.md [hunk 2]
```diff
@@ -73,6 +73,10 @@ Download a release of the toolchain for your platform from [Github
...
+
+Install appropriate latest supported Microsoft Visual C++ Redistributable package, such as from [Microsoft Visual C++ Redistributable latest supported downloads](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170)
```
Comment (voltur01) [inline README.md:78]: Add a full stop at the end?

**Review by voltur01 — COMMENTED** [review]
Thank you for adding this! Please consider some minor comments, otherwise looks good to me.

File: README.md (hunk 1 above)
Comment (vrukesh) [reply README.md]: Fixed in latest version of patch.

Comment (vrukesh) [reply README.md]: Fixed in latest version of patch.

Comment (vrukesh) [reply README.md]: Fixed in latest version of patch.

Comment (vrukesh) [reply README.md]: Fixed in latest version of patch.

File: README.md (hunk 2 above)
Comment (vrukesh) [reply README.md]: Fixed in latest version of patch.

File: README.md (hunk 1 above)
Comment (vrukesh) [reply README.md]: Fixed in latest version of patch.

Comment (vrukesh) [reply README.md]: Fixed in latest version of patch.

**Review by voltur01 — APPROVED** [review]
Thank you for the update!

**Pull request merged**